### PR TITLE
python-cffi: update to 1.5.2

### PR DESCRIPTION
**Pull request by jefferyto**
Signed-off-by: Jeffery To <jeffery.to@gmail.com>

### CODE REVIEW

File: lang/python-cffi/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cffi
-PKG_VERSION:=1.5.0
+PKG_VERSION:=1.5.2
 PKG_RELEASE:=2
```
Comment (yousong) [inline lang/python-cffi/Makefile]: PKG_RELEASE should be reset to 1

**Comment by hnyman** [comment]
@yousong 
Sorry, I did not notice the comment before I merged it.

@jefferyto 
Next time, please remember to reset the PKG_RELEASE to 1 when PKG_VERSION increases.

**Comment by jefferyto** [comment]
Sorry about that :bow:

**Comment by yousong** [comment]
Well, we can increase PKG_RELEASE from 2 and reset it to 1 at the next version bump :)

**Pull request merged**